### PR TITLE
PS-8093 Wrong documentation for rocksdb_perf_context_level (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version

### CODE REVIEW

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -2174,7 +2174,25 @@ for the MyRocks MTR test suites.
...
 Default value is ``0``.
-Allowed range is up to ``4``.
+Allowed range is up to ``5``.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value
+     - Description
+   * - 0
```
Comment (kamil-holubicki) [inline doc/source/myrocks/variables.rst:2184]: It is an interesting setting. "Unknown". In implementation it acts as 'disable (1)', but when the value of the variable is queried it indeed returns unknown (0). Moreover, unknown is the default...

Why just don't have values from 1(disabled (default)) to 5?

**Review by kamil-holubicki — APPROVED** [review]

File: doc/source/myrocks/variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst:2184]: Discussed in Slack.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Comment by patrickbirch** [comment]
I have closed this pull request because the documentation has been moved to another repository.

**Pull request closed without merging**